frontend/src/firebase/firebase.ts

```diff
@@ -19,4 +19,14 @@ export const initFirebase = () => {
 
 const { app, auth } = initFirebase();
 
-export { app, auth };
+const checkEmailExists = async (email: string) => {
+  try {
+    const methods = await fetchSignInMethodsForEmail(auth, email);
```

Comment:
I think this function is now deprecated in Firebase. Not sure if it still works. Don't need to change anything for now.